uk.co.compucorp.civicrm.hrcore/hrcore.php

```diff
@@ -297,3 +322,22 @@ function _hrcore_add_js_session_vars() {
     'contact_id' => CRM_Core_Session::getLoggedInContactID()
   ]);
 }
+
+/**
+ * Checks if an extension is installed or enabled
+ *
+ * @param string $key
+ *   Extension unique key
+ *
+ * @return boolean
+ */
+function _hrcore_isExtensionEnabled($key)  {
```

Comment:
Please use the [ExtensionHelper](https://github.com/civicrm/civihr/blob/staging/uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Helper/ExtensionHelper.php) for this